crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -549,11 +550,11 @@ pub enum RenderMeshInstanceGpuQueue {
     /// The version of [`RenderMeshInstanceGpuQueue`] that omits the
     /// [`MeshCullingData`], so that we don't waste space when GPU
     /// culling is disabled.
-    CpuCulling(Vec<(Entity, RenderMeshInstanceGpuBuilder)>),
+    CpuCulling(Vec<(RenderEntity, RenderMeshInstanceGpuBuilder)>),
     /// The version of [`RenderMeshInstanceGpuQueue`] that contains the
     /// [`MeshCullingData`], used when any view has GPU culling
     /// enabled.
-    GpuCulling(Vec<(Entity, RenderMeshInstanceGpuBuilder, MeshCullingData)>),
+    GpuCulling(Vec<(RenderEntity, RenderMeshInstanceGpuBuilder, MeshCullingData)>),
```

Comment:
Is this strictly necessary? Entity always refers to an entity in the world that you are in (in this case the render world), so they are already render entities.

Reply:
It's not necessary, but when you look at a datatype out of context it's really hard to figure out which world the entities are from if it's not documented. When you're doing extract and sync systems you have access to entities from both worlds, and you could reasonably need to store either. 

I know `RenderEntity` wasn't intended for this, but using it as a newtype wrapper communicates intent really clearly, and because it's used from an extract system you have the `RenderEntity` component at hand and can just use it directly.